lexic.el

```diff
@@ -340,7 +340,7 @@ Using `lexic-current-dictionary-list' and `lexic-dictionary-path'."
 (defun lexic-return-from-lexic ()
   "Bury lexic buffer and restore the previous window configuration."
   (interactive)
-  (kill-process (get-process lexic-process-name))
+  (ignore-errors (kill-process (get-process lexic-process-name)))
```

Comment:
unrelated fix of small annoyance - sometimes pressing 'q' did not bury the buffer, because killing the already dead buffer  caused an error